easybuild/easyconfigs/m/MAGeCK/MAGeCK-0.5.9.4-foss-2021a.eb

```diff
@@ -0,0 +1,34 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonPackage'
+
+name = 'MAGeCK'
+version = '0.5.9.4'
+
+homepage = "https://sourceforge.net/p/mageck/wiki/Home/"
+description = """Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify
+ important genes from the recent genome-scale CRISPR-Cas9 knockout screens (or GeCKO) technology. MAGeCK is developed
+ by Wei Li and Han Xu from Dr. Xiaole Shirley Liu's lab at Dana-Farber Cancer Institute, and is being actively updated
+ by Wei Li lab from Children's National Medical Center."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['42875b308f2d9ee38f083fb1020d7d48fca29e2100fdb09ced1969b738b8c939']
+
+# remove the shipped mageckGSEA and RRA binaries so that the Python install will build them
```

Comment:
Maybe say that this is for RPATH to work correctly, and refer to this PR. Without such comments, it's a bit unclear why you'd want to do this.

Reply:
bf90f29